File: 3172-DivisibleAndNonDivisibleSumsDifference/3172-DivisibleAndNonDivisibleSumsDifference.py
# Last updated: 01/09/2025, 00:58:29
class Solution:
    def differenceOfSums(self, n: int, m: int) -> int:
        p = 0
        q = 0
        for i in range(1 , n + 1):
            if i % m == 0:
                p += i
            else:
                q += i
        return q - p
        

File: 3172-DivisibleAndNonDivisibleSumsDifference/3172-DivisibleAndNonDivisibleSumsDifference.cpp
// Last updated: 01/09/2025, 00:58:33
class Solution {
public:
    int differenceOfSums(int n, int m) {
        int p=0,q=0;
        for(int i=1;i<=n;i++){
            if(i%m==0) p+=i;
            else if(i%m!=0) q+=i;
        }
    return q-p;
    }
};